packages/components/pages/f-contact-preferences/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-contact-preferences",
   "description": "Fozzie Contact Preferences - Fozzie user contact preferences form component",
-  "version": "0.5.2",
+  "version": "0.5.3",
```

Comment:
Should be a minor bump (patches are for fixes/tiny changes)

Reply:
Not ready to publish yet so keeping as minor

Reply:
If it won't be published you could always do a `Latest` changelog entry, don't bump the version and then stick `#trivial` in the PR title to get around the danger checks

Reply:
Get that it's not ready to be published but we should still follow semver for version bumps, as it gives an indication of the size of the changes (which is useful when looking to review too). A patch release should be a couple of line change – minor is any minor functionality change.


Or as Xander said, if you're looking to group changes together, then use latest in the changelog to avoid having to bump the package version.

Reply:
I have a pr to follow this one in which I will bump to version 0.6.0 and publish 👍